src/components/Messages/MessageList/index.js

```diff
@@ -147,14 +155,39 @@ export class MessageList extends MemoizedComponent {
 				);
 			}
 
+			const isMyMessage = (msg) => uid && msg?.u && uid === msg?.u._id;
```

Comment:
Can we move functions related to sequence to some util/helper file? To make it more readable

Reply:
Yes totally makes sense, Will do it
Thanks!